wiki/troubleshooting.md

```diff
@@ -17,6 +17,14 @@ To run SFC, from am elevated command prompt run `sfc /scannow`. It is recommend
 
 For more information about SFC, check [this page](https://support.microsoft.com/en-us/help/929833/use-the-system-file-checker-tool-to-repair-missing-or-corrupted-system)
 
+## CHKDSK
+
+This utility checks the file system and file system metadata of a volume for errors.
+To use CHKDSK, open CMD (aka. Command Prompt) with administrator privileges and type `<volume> /f`.
```

Comment:
```suggestion
To use CHKDSK, open the Command Prompt with Administrator Privileges and type `chkdsk <volume> /f`.
```

Reply:
There's no need to capitalize "Administrator Privileges," and "the" isn't strictly necessary.
```suggestion
To use CHKDSK, open Command Prompt with administrator privileges and type `chkdsk <volume> /f`.
```
Alternatively:
```suggestion
To use CHKDSK, open Command Prompt (be sure to select 'run as administrator') and type `chkdsk <volume> /f`.
```
Either of these is fine.